ModulBot/Commands/TuneAdmin.cs

```diff
@@ -0,0 +1,117 @@
+﻿using ModulBot.Database.PostgresRepositories;
+using ModulBot.Interfaces;
+using ModulBot.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Telegram.Bot.Types;
+using ModulBot.Attributes;
+
+namespace ModulBot.Commands
+{
+    [NotShowInHelp]
+    [Serializable]
+    public class TuneAdmin : ITool
+    {
+        public string Description { get; set; }
+        public List<string> CommandsName { get; set; }
+        public bool IsAdmin { get; set; }
+        private static int step = 0;
+        private static int slctUserId = 0;
+
+        public async void Run(Message message)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 22 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS3776)